appcenter-analytics/ios/appcenter-analytics.podspec

```diff
@@ -10,10 +10,9 @@ Pod::Spec.new do |s|
   s.homepage          = package['homepage']
   s.documentation_url = "https://docs.microsoft.com/en-us/appcenter/"
 
-  s.author           = { 'Microsoft' => 'appcentersdk@microsoft.com' }
-
-  s.source = { :git => "https://github.com/Microsoft/AppCenter-SDK-React-Native.git" }
-  s.source_files = "appcenter-analytics/ios/AppCenterReactNativeAnalytics/*.{h,m}"
+  s.author            = { 'Microsoft' => 'appcentersdk@microsoft.com' }
+  s.source            = { :git => "https://github.com/Microsoft/AppCenter-SDK-React-Native.git" }
+  s.source_files      = "AppCenterReactNativeAnalytics/**/*.{h,m}"
```

Comment:
Was that required that you replace some part of the paths with stars?

Reply:
Not required, however, this way it included sub-folders as well.